### PR TITLE
ozr-5x tune preset BF 4.5

### PR DESCRIPTION
**Pull request by limonspb**
5" frame racing tune by Dindrones and Benjamin Freelove

### CODE REVIEW

**Review by sugaarK — COMMENTED** [review]
3 blade props hit the fundamental  and the  second harmonic  pretty hard. Turning off the last one doesn’t save any thing in terms of latency vs the second. Use the rpm weight to reduce the first harmonic. This is a much better approach to tuning 4.5.

File: presets/4.5/tune/ozr_5x.txt [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+set rpm_filter_fade_range_hz = 100
+
+# -- Misc --
+set yaw_spin_recovery = AUTO
+set thrust_linear = 0
+set throttle_boost = 0
+set ez_landing_limit = 5
+
+# -- iTerm  --
+set iterm_relax_cutoff = 20
```
Comment (sugaarK) [inline presets/4.5/tune/ozr_5x.txt:58]: I’d make this 45 as well, tracks better

**Comment by limonspb** [comment]
Waiting for Benjamin to reply :)

File: presets/4.5/tune/ozr_5x.txt [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+#$ DESCRIPTION: <h1>DinDrones OZR-5X tune</h1>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: This 6S racing tune is based on Karate Race 6s 5" (non spicy) by KarateBrot with very minor changes.
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: Tested on DinDrones OZR-5X frame with Foxeer F722 v4 MPU6000 flight controller and default ESC settings.
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION:
+#$ DESCRIPTION: ## Things to note:
+#$ DESCRIPTION: - **YOU HAVE TO USE RPM FILTERING WITH THIS TUNE!** Failure to do so might result in fire ðŸ”¥
```
Comment (KarateBrot) [inline presets/4.5/tune/ozr_5x.txt:18]:
```suggestion
#$ DESCRIPTION: - **YOU HAVE TO USE RPM FILTERING WITH THIS TUNE!** Failure to do so might result in fire 🔥
```

File: presets/4.5/tune/ozr_5x.txt [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+#$ CATEGORY: TUNE
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: ozr, dindrones, 6s, 5 inch, 5", racing, 5in, din
+#$ AUTHOR: Benjamin Freelove
+
+#$ PARSER: MARKED
+
+#$ DESCRIPTION: <h1>DinDrones OZR-5X tune</h1>
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: This 6S racing tune is based on Karate Race 6s 5" (non spicy) by KarateBrot with very minor changes.
```
Comment (KarateBrot) [inline presets/4.5/tune/ozr_5x.txt:12]:
```suggestion
#$ DESCRIPTION: This 6S racing tune is based on Karate Race 6s 5" (non spicy) by sugarK with very minor changes.
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]

File: presets/4.5/tune/ozr_5x.txt (hunk 2 above)
Comment (haslinghuis) [reply presets/4.5/tune/ozr_5x.txt:18]: `ðŸ¤£`

Comment (limonspb) [reply presets/4.5/tune/ozr_5x.txt:18]: > `ðŸ¤£`

Ÿð¤ðŸ¤£

Comment (haslinghuis) [reply presets/4.5/tune/ozr_5x.txt:18]: âœŒ

**Comment by limonspb** [comment]
@sugaarK 
updated based on your suggestions
And removed some values that are = to 4.5 defaults

**Review by sugaarK — APPROVED** [review]

**Pull request merged**